pandas/tests/series/methods/test_reset_index.py

```diff
@@ -163,6 +163,12 @@ def test_reset_index_inplace_and_drop_ignore_name(self):
         expected = Series(range(2), name="old")
         tm.assert_series_equal(ser, expected)
 
+    def test_reset_index_drop_infer_string(self):
+        # GH#56160
+        ser = Series(["a", "b", "c"], dtype=object)
```

Comment:
This test doesn't need to involve the `infer_string` option?

Reply:
Oh yikes, added It, thx